### DISCUSSION TITLE
run scoop error `not recognized as the name of a script file. `

### DISCUSSION THREAD

**Discussion by scil** [ScoopInstaller/Scoop#5024] [Show and tell]
suddenly, i encounter this error when run `sccop`

```
> scoop list

The argument ' 'D:\A\Scoop\apps\scoop\current\bin\scoop.ps1'  list;exit $LASTEXITCODE' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again
```

I found the error is from `Scoop\shims\scoop.cmd`.

I change the line  
` powershell -noprofile -ex unrestricted "& 'D:\A\Scoop\apps\scoop\current\bin\scoop.ps1'  %args%;exit $LASTEXITCODE"`
to   
`powershell -noprofile -ex unrestricted "D:\A\Scoop\apps\scoop\current\bin\scoop.ps1"  %args%`  

Now `scoop`  works well.

**Comment by rashil2000** [comment]
You can delete the scoop.cmd file and run `scoop update`, it will get generated again.
